yocto-dev/build_conf/bblayers.conf

```diff
@@ -0,0 +1,54 @@
+LCONF_VERSION = "7"
+
+BBPATH = "${TOPDIR}"
+BSPDIR := "${@os.path.abspath(os.path.dirname(d.getVar('FILE', True)) + '/../..')}"
+
+BBFILES ?= ""
+BBLAYERS = " \
+  ${BSPDIR}/sources/poky/meta \
+  ${BSPDIR}/sources/poky/meta-poky \
+  \
+  ${BSPDIR}/sources/meta-openembedded/meta-oe \
+  ${BSPDIR}/sources/meta-openembedded/meta-multimedia \
+  ${BSPDIR}/sources/meta-openembedded/meta-python \
+  \
+  ${BSPDIR}/sources/meta-freescale \
+  ${BSPDIR}/sources/meta-freescale-3rdparty \
+  ${BSPDIR}/sources/meta-freescale-distro \
+"
+
+# i.MX Yocto Project Release layers
+BBLAYERS += "${BSPDIR}/sources/meta-imx/meta-bsp"
+BBLAYERS += "${BSPDIR}/sources/meta-imx/meta-sdk"
+BBLAYERS += "${BSPDIR}/sources/meta-imx/meta-ml"
+BBLAYERS += "${BSPDIR}/sources/meta-imx/meta-v2x"
+BBLAYERS += "${BSPDIR}/sources/meta-nxp-demo-experience"
+
+BBLAYERS += "${BSPDIR}/sources/meta-browser/meta-chromium"
+BBLAYERS += "${BSPDIR}/sources/meta-clang"
+BBLAYERS += "${BSPDIR}/sources/meta-openembedded/meta-gnome"
+BBLAYERS += "${BSPDIR}/sources/meta-openembedded/meta-networking"
+BBLAYERS += "${BSPDIR}/sources/meta-openembedded/meta-filesystems"
+BBLAYERS += "${BSPDIR}/sources/meta-qt6"
+BBLAYERS += "${BSPDIR}/sources/meta-virtualization"
+BBLAYERS += "${BSPDIR}/sources/meta-bsp-imx9"
+BBLAYERS += " \
+	${BSPDIR}/sources/meta-compulab \
+	${BSPDIR}/sources/meta-compulab-bsp/meta-desktop \
+	${BSPDIR}/sources/meta-compulab-bsp/meta-multimedia \
+	${BSPDIR}/sources/meta-compulab-bsp/meta-graphics \
+	${BSPDIR}/sources/meta-compulab-bsp/meta-utils \
+	${BSPDIR}/sources/meta-compulab-bsp/meta-bsp \
+	${BSPDIR}/sources/meta-compulab-bsp/meta-devtool \
+	${BSPDIR}/sources/meta-compulab-uefi \
+"
+
+# additional layers for Ice Cream
```

Comment:
Remove code name from comment. Mention what we actually need these ROS layers for e.g. the following dependencies needed to run Kimera-VIO can be found in <layer>.